src/features/VoltageConfiguration/VoltageConfiguration.tsx

```diff
@@ -21,24 +21,36 @@ interface VoltageConfigurationProps {
     pmicPort: number;
     voltageMin: number;
     voltageMax: number;
+    pmicPortLabel?: string;
+    pmicPortDescription?: string;
 }
 
 const VoltageConfiguration = ({
     pmicPort,
     voltageMin,
     voltageMax,
+    pmicPortLabel,
+    pmicPortDescription,
 }: VoltageConfigurationProps) => {
     const dispatch = useDispatch();
     const voltage = useSelector(getPmicConfigValue(pmicPort)) ?? voltageMin; // Default to voltageMin
 
+    const label = pmicPortLabel ?? `Voltage port ${pmicPort}`;
+    const description =
+        pmicPortDescription ??
+        `Set voltage for PMIC port ${pmicPort} (${label})`;
+
     return (
         <Card
             title={
                 <div className="tw-flex tw-content-between">
-                    <span>Voltage</span>
+                    <span>{label}</span>
                 </div>
             }
         >
+            <div className="tw-flex tw-gap-1">
+                <p>{description}</p>
+            </div>
```

Comment:
Maybe this comment goes for this file in general, but are you sure about only showing the values conditionally instead of also wrapping each div/outer element in a {value && (element...) }?

Reply:
Might remove the elements all together, yes. I'll add that and possibly change it later.